modular_skyrat/modules/company_imports/code/armament_datums/akh_frontier.dm

```diff
@@ -21,6 +21,28 @@
 /datum/armament_entry/company_import/akh_frontier/basic/compact_drill
 	item_type = /obj/item/pickaxe/drill/compact
 
+// Wearable stuff, not inclusive of all clothing because you're supposed to print those
+
+/datum/armament_entry/company_import/akh_frontier/wearables
+	subcategory = "Wearable Equipment"
+
+/datum/armament_entry/company_import/akh_frontier/wearables/hazard_mod
+	name = "Frontier Hazard Protective MOD Control Unit"
+	description = "The pinnacle of frontier cheap technology. Suits such as these are made specifically for the rare emergency that creates a hazard \
+		environment that other equipment just can't quite handle. Often, these suits are able to protect their users \
+		from not only electricity, but also radiation, biological hazards, other people, so on. This suit will not, \
+		however, protect you from yourself."
+	item_type = /obj/item/mod/control/pre_equipped/frontier_colonist
+	cost = PAYCHECK_COMMAND * 6.5
```

Comment:
This is ridiculously cheap compared to all of the other available MODsuit options, for reference the Engineering MODsuit costs about 2600 in the regular Cargo console orders.

Reply:
There are enough downsides piled onto this suit that I think anyone doing stuff that you would want, say, an engineering modsuit for would rather have the engi mod over this regardless of how cheap this is.